catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/WorkspacePreIngestPlugin.java

```diff
@@ -79,14 +120,16 @@ public CreateRequest process(CreateRequest request)
                 .filter(workspace -> StringUtils.isEmpty(workspace.getOwner()))
                 .collect(Collectors.toList());
 
-        if (!workspaces.isEmpty() && StringUtils.isEmpty(email)) {
+        if (!workspaces.isEmpty() && !owner.isPresent()) {
+            alertMissingClaim(ownerAttr);
             throw new StopProcessingException(String.format(
-                    "Cannot create workspace. Subject with name '%s' not permitted because they have no email.",
-                    getSubjectName()));
+                    "Cannot create workspace. Subject with name '%s' not permitted because they are missing the '%s' attribute.",
```

Comment:
I'm assuming this gets returned to the client. Do we actually need to tell them what the subject is missing, or can we just simply say that they are not permitted. We can always log which attribute is missing and could provide a "The administrator can check the logs for this ID." Sort of statement.

Reply:
This exception will get caught by the query endpoint which will return a generic error message.